kubernetes-artifacts/prometheus/gpu-nodeExpoter.yml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: node-gpu-exporter
+spec:
+  selector:
+    matchLabels:
+      app: node-gpu-exporter
+  template:
+    metadata:
+      labels:
+        app: node-gpu-exporter
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: aliyun.accelerator/nvidia_count
```

Comment:
Maybe it is not universal.

Reply:
You are right, I updated docs, user need to label their GPU node manually.